packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -608,10 +608,10 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
                 backgroundColor: highlight.backgroundColor
             };
         }
-        const createChildren = (fragment: TreeDecoration.CaptionHighlight.Fragment) => {
+        const createChildren = (fragment: TreeDecoration.CaptionHighlight.Fragment, index: number) => {
```

Comment:
Before I changed this, I received React errors about array children without keys. I'm not certain why that error doesn't occur in other trees, but if anyone knows how to get away with it in the SCM too, I'm happy to use that rather than changing this function. That said, I don't anticipate that this should cause problems.